deployments/dev/environment.py

```diff
@@ -55,127 +55,345 @@ def env() -> Mapping[str, Optional[str]]:
         'AZUL_DEPLOYMENT_STAGE': 'dev',
 
         'AZUL_CATALOGS': json.dumps({
-            name: dict(atlas=atlas,
-                       internal=internal,
-                       plugins=dict(metadata=dict(name='hca'),
-                                    repository=dict(name='tdr')))
-            for name, atlas, internal in [
-                ('dcp2', 'hca', False),
-                ('it2', 'hca', True),
+            f'{prefix}{name}': dict(atlas=atlas,
+                                    internal=internal,
+                                    plugins=dict(metadata=dict(name='hca'),
+                                                 repository=dict(name='tdr')),
+                                    sources=sources)
+            for name, atlas, sources in [
```

Comment:
No point in looping over a singleton.

Reply:
I was hoping to have this in a state where we can easily add the `lungmap` catalogs back in when we work on https://github.com/DataBiosphere/azul/issues/3489 but i don't think we can have this and each snapshot on a line.